### PR TITLE
DELETE /api/users/:userid/boards/:id  endpoint

### LINKED ISSUE #62: Backend: Endpoint to delete a specific board - DELETE /api/users/:userid/boards/:id
## User story
**Who**: As a developer
**What**: Owner can delete the board of a specific user
**Why**: So that we can delete a board of a specific user, if the user wishes to delete a board.
## Acceptance criteria
The endpoint should delete the particular board of the user.

Create the DELETE endpoint: DELETE /api/users/:userid/boards/:id
Should work like this: /api/users/13/boards/7 where 13 is the user (creator / owner) and 7 is the id of the board
This endpoint should be documented in Swagger /api/documentation

Screenshot from 2021-07-17 12-35-15_2

## Implementation details

User-id should be the same user that client is authenticated with and if not the same or If the user is not the owner / creator of the board, access deleting the board should be denied.

### PR DESCRIPTION
**Pull request by Bhuvaneshwari29**
# Description

Endpoint to delete a specific board

Fixes #62 

# How to test?

we can test it by swagger when we insert some data in boards table and user table and delete it by some specific id.
(for example
insert into users(id,fullname, email, firebaseUId) values (1,'Priya', 'priya@gmail.com', 123);
insert into users(id,fullname, email, firebaseUId) values (2,'sanju', 'sanju@gmail.com', 456);

INSERT INTO boards(id,creatorId,createdOn,title,deadline,isDeleted,banner) 
VALUES (1,1,'2021-07-01 10:10:10','ELECTION','2021-07-05 10:10:10',false,'ELECTIONBANNER');

INSERT INTO boards(id,creatorId,createdOn,title,deadline,isDeleted,banner) 
VALUES (2,2,'2021-07-01 10:10:10','ELECTION','2021-07-05 10:10:10',false,'BANNER'); )

# Checklist

- [x] I have performed a self-review of my own code
- [x ] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [x] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [x] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Comment by Bhuvaneshwari29** [comment]
Hi Niels, @senner007 Authorization check added and checked in Swagger. Works fine. Can we merge?

**Comment by senner007** [comment]
Hi
What's the status on this?

File: README.md [hunk 1]
```diff
@@ -12,7 +11,6 @@
   <img  height=230px src="https://user-images.githubusercontent.com/6642037/124296829-afecbf80-db5a-11eb-9dbc-d92b588e1f42.png" alt="Project Homepage"></a>
 </p>
 
-
```
Comment (senner007) [inline README.md:15]: Hi.

This change to readme has to be reverted. 
git checkout develop readme.md

**Review by senner007 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Bhuvaneshwari29) [reply README.md:15]: Hi Niels, will revert it .

**Comment by Bhuvaneshwari29** [comment]
> Hi
> What's the status on this?

Hi Niels,
I added the authorization part and tested if everything works fine. Its works fine. Now will revert the read me part as you mentioned.

**Comment by Bhuvaneshwari29** [comment]
@senner007 Hi Niels, Reverted the Readme file

**Review by senner007 — CHANGES_REQUESTED** [review]
Please reorder the displaced swagger comments

**Comment by Bhuvaneshwari29** [comment]
> Please reorder the displaced swagger comments

@senner007 Hi Niels , Reordered the displaced swagger comments

**Comment by senner007** [comment]
> > Please reorder the displaced swagger comments
> 
> @senner007 Hi Niels , Reordered the displaced swagger comments

Please make sure the imports and router uses are correct. Right now you have used candidates router twice. When you request a review you should go through the code and verify its correctness

**Comment by senner007** [comment]
You also have a merge conflict, I see

**Comment by Bhuvaneshwari29** [comment]
> You also have a merge conflict, I see

@senner007  Hi Niels, I resolved now. Please check

**Review by senner007 — APPROVED** [review]
👍

**Pull request merged**